docs/common/guides/jlink_image.adoc

```diff
@@ -0,0 +1,207 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2020 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Custom Runtime Images with `jlink`
+
+// :description: Helidon Custom Runtime Images
+// :keywords: helidon, guide, jlink, image
+// :helidon-uc-edition: MP-or-SE
+// :helidon-lc-edition: mp-or-se
+// :docker-image-size: some-number
+// :docker-multi-stage-url: https://docs.docker.com/develop/develop-images/multistage-build/
+
+This guide describes how to build a custom runtime image for your Helidon application
+using Helidon's support for the JDK's `jlink` tool.
+
+== Introduction
+
+JDK 9 introduced the https://docs.oracle.com/en/java/javase/11/tools/jlink.html[`jlink`]
+command that supports assembling a set of modules and their dependencies into a custom
+runtime image. The `helidon-mave-plugin` has support for easily creating a custom runtime image for your
+Helidon application resulting in a smaller, better performing runtime.
+
+In this guide you will learn how to build a custom runtime image locally on your machine,
+as well as using Docker.
+
+== What You Need
+
+|===
+|About 10 minutes
+| <<about/03_prerequisites.adoc,Helidon Prerequisites>>
+|===
+
+== Verify Java 11 or Newer
+
+[source,bash]
+----
+$JAVA_HOME/bin/jlink --version
```

Comment:
This might give the impression that we're going to use jlink directly; how about just `java -version`.

Reply:
fixed